mkosi/config.py

```diff
@@ -3075,6 +3076,15 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         parse=config_make_path_parser(required=False),
         help="Set the path used to store forwarded machine journals",
     ),
+    ConfigSetting(
+        dest="bump_script",
+        metavar="PATH",
+        section="Host",
```

Comment:
Should be same section as the image version so `Output`

Reply:
Done. Had it as `Host` mostly because it is something entirely run on the host and doesn't really have much to do with the actual output (just to explain my thought process).